### DISCUSSION TITLE
你们不管 vim copilot 了吗

### DISCUSSION THREAD

**Discussion by kaneruan** [community/community#135829] [Copilot Conversations]
根本无法用，经常性没有 completition 出来，产品体验真的很差。还是个付费的。

**Comment by Akash1134** [comment]
Hi @kaneruan , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
